openrr-gui/src/velocity_sender.rs

```diff
@@ -411,6 +414,9 @@ where
                     state.update_value(state.value.abs() * sign as f64);
                 }
             }
+            Message::Retained => {
+                self.move_base.send_velocity(&latest_velocity).unwrap();
```

Comment:
L422-L424 also sends velocity, so I think you can remove this `send_velocity` and L339's` current_velocity`.

Reply:
Thank you for comment. I update as noted above.